<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Administration</class>
 <widget class="QWidget" name="Administration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教务处管理</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>631</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_selectON">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>学生选课/退选开放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_selectOFF">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>学生选课/退选关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_add">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_del">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_search">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_refresh">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_searchline">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_logout">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>登出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_view">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>选课学生显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_TeacherManger">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>教师管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_teacherCus">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>教师排课信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
